.github/workflows/tests.yml

```
@@ -160,6 +172,76 @@ jobs:
         flags: unittests
         name: codecov-umbrella
 
+  # Playwright E2E Tests
+  playwright-tests:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.should-run-tests == 'true'
+
+    steps:
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)